docs: documentation for jest exit error. resolve spellchecks warning … (#232)
askui/askui: docs/docs/general/07-Troubleshooting/windows.md

# Windows

## askui UI Controller 

On Windows, the askui UI Controller may not terminate after test execution. This may lead to problems when using the askui UI Controller in a pipeline like a pipeline's run not coming to an end. There are two ways to deal with this issue:

- force termination of the askui UI Controller. The `UiController.stop()` takes an optional boolean argument which is set to `false` by default. If we pass `true` to the `UiController.stop()` method, e.g., `uiController.stop(true)`,  we kill the askui UI Controller process. 

> - force Jest to exit as described on our [Jest-Troubleshooting page](jest.md). This option terminates all processes which were set up by test code and didn't close properly. The first option explicitly stops the askui UI Controller. 

Reviewer: I think there are still some open questions that I would have as a reader: Why are there two ways with dealing with this issue? When and why should I use which one? 

Can we explain this or, maybe, can we even remove one of the options?

```typescript
await uiController.stop(true);

> ```

Reviewer: Code example seems out of place...

# Windows

> ## askui Controller 

Reviewer: ```suggestion
## askui UI Controller 
```

> On windows the problem occurs that the askui Controller doesn't terminate after test execution.

Reviewer: ```suggestion
On Windows, the askui UI Controller may not terminate after test execution. This may lead to problems when using the askui UI Controller in a pipeline like a pipeline's run not coming to an end. There are 2 ways to deal with this issue:

- configuring Jest to force the exit of child processes, or,
- ...
```

Jest can't execute if processes are still running. If you are using askui in a CI/CD pipeline
this can prevent the pipeline to proceed.
[Here](jest.md) is already a possible solution where you force Jest to exit.

> But you could also try to stop the askui Controller. This can be done by passing `true` to

Reviewer: ```suggestion
But you could also try to stop the askui UI Controller. This can be done by passing `true` to
```

the stop method:

```typescript
await uiController.stop(true);

> ```

Reviewer: Can you explain what `true` means? What happens if the user passes that?

# Windows

## askui UI Controller 

On Windows, the askui UI Controller may not terminate after test execution. This may lead to problems when using the askui UI Controller in a pipeline like a pipeline's run not coming to an end. There are two ways to deal with this issue:

- you can force Jest to exit as described on our [Jest-Troubleshooting page](jest.md)

> - try to stop the askui UI Controller. The stop method of the class `UiController` has an optional `forceStop` parameter. If we pass `true` to the stop method we kill the process of the askui UI Controller port and force a termination.

Reviewer: Can you explain how this is different from the first option? 

```suggestion
- force termination of the askui UI Controller. The `UiController.stop()` takes an optional boolean argument which is set to `false` by default. If we pass `true` to the `UiController.stop()` method, e.g., `uiController.stop(true)`,  we kill the askui UI Controller process.
```

"killing" already implies forcing termination
